### PR TITLE
main: release 0.16.0

### PR DESCRIPTION
**Pull request by aykevl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+
+* **command-line**
+  - add initial support for LLVM 11
+  - `build`: improve support for GOARCH=386 and add tests
+  - `gdb`: add support for qemu-user targets
+  - `test`: support non-host tests
+  - `test`: add support for -c and -o flags
+  - `test`: implement some benchmark stubs
+* **compiler**
+  - `builder`: improve detection of clang on Fedora
```
Comment (QuLogic) [inline CHANGELOG.md:13]: Can we include #1424 for this?

Comment (aykevl) [reply CHANGELOG.md:13]: Done.

**Comment by aykevl** [comment]
I included the latest changes, so this release is now ready from my point of view.

File: CHANGELOG.md (hunk 1 above)
Comment (QuLogic) [reply CHANGELOG.md:13]: Thanks!

**Comment by aykevl** [comment]
@deadprogram it appears that TinyHCI is stuck.

**Comment by deadprogram** [comment]
OK TinyHCI fixed.

**Comment by deadprogram** [comment]
@aykevl this needs to be updated for some of the commits that have landed since it was created.

Are there any other commits that need to merge now before the actual release?

**Comment by aykevl** [comment]
Rebased and updated.



> Are there any other commits that need to merge now before the actual release?

I don't think so. And I'd rather get the release out and fix things the next release than delay this release, to be honest.

**Comment by deadprogram** [comment]
Agreed! Now merging.

**Pull request merged**